build/src/src/modules/legacy/removeLegacyBindVolume.ts

```diff
@@ -10,25 +10,24 @@ const newBindVol = "dncore_binddnpdappnodeeth_bind";
 /**
  * After core version 0.2.30 there will be an orphan volume of the
  * DNP_BIND which should be removed for asthetic versions
+ *
+ * After core version 0.2.34 the new volume will become orphan and
+ * must be removed for the bind state to be reset
  */
 export async function removeLegacyBindVolume(): Promise<void> {
   const volumes = await dockerVolumesList();
   const volumeExists = (name: string): boolean =>
     volumes.some(vol => vol.Name === name);
 
-  // nuevo-> dncore_binddnpdappnodeeth_bind
-  // viejo-> dncore_binddnpdappnodeeth_data
-  // Remove all packages that are using the volume to safely move it
+  for (const volName of [oldBindVol, newBindVol]) {
+    if (volumeExists(volName)) {
+      const users = await shell(`docker ps -aq --filter volume=${oldBindVol}`);
+      if (users)
+        throw Error(`legacy BIND volume ${volName} has users: ${users}`);
 
-  if (volumeExists(oldBindVol) && volumeExists(newBindVol)) {
-    const usersOfOld = await shell(
-      `docker ps -aq --filter volume=${oldBindVol}`
-    );
-    if (usersOfOld)
-      throw Error(`legacy BIND volume ${oldBindVol} has users: ${usersOfOld}`);
-
-    // Delete only if has no users
-    await dockerVolumeRm(oldBindVol);
-    logs.info(`Removed legacy BIND volume ${oldBindVol}`);
+      // Delete only if has no users
+      await dockerVolumeRm(oldBindVol);
+      logs.info(`Removed legacy BIND volume ${oldBindVol}`);
+    }
```

Comment:
Generalize the existing removeLegacyBindVolume to kill both BIND volumes if the exist and have no users